access-control-app/Procfile

```diff
@@ -0,0 +1 @@
+web: java -classpath /app/cmr-access-control-app-0.1.0-SNAPSHOT-standalone.jar cmr.access-control-app.runner
```

Comment:
Our Procfile is currently:

```web: java \$JVM_OPTS -cp target/${app_filename} clojure.main -m cmr.${app_name}.runner```

I'm thinking we probably want to keep using the same Procfile when we deploy to NGAP from within Bamboo. I wouldn't put these individual Procfiles in yet, let's just commit the Dockerfiles.